Content.Server/GameObjects/Components/Singularity/ContainmentFieldGeneratorComponent.cs

```diff
@@ -64,6 +65,7 @@ private void TryPowerConnection(ref Tuple<Direction, ContainmentFieldConnection>
         }
 
         private PhysicsComponent? _collidableComponent;
+        private PointLightComponent? _pointLightComponent;
```

Comment:
use componentdependencies pls

Reply:
I'll refactor both of these, is that acceptable?

Reply:
Assuming yes, resolved in latest commit